content/codepipeline/configmap.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Modify aws-auth ConfigMap"
+date: 2018-10-087T08:30:11-07:00
+weight: 11
+draft: true
+---
+
+Now that we have the IAM role created, we are going to add the role to the [aws-auth ConfigMap](https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html)
+for the EKS cluster.
+
+Once the ConfigMap includes this new role, kubectl in the CodeBuild stage of the pipeline will be able to interact with the EKS cluster via the IAM role.
+
+```
+ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+ROLE="    - rolearn: arn:aws:iam::$ACCOUNT_ID:role/EksWorkshopCodeBuilKubectldRole\n      username: build\n      groups:\n        - system:masters"
```

Comment:
Let's put some carriage returns between the separate commands to increase readability.